src/commands/deployPackageCommand.ts

```diff
@@ -91,9 +91,17 @@ export class DeployPackageCommand implements ICommand {
         this.diags.clear();
         this.handleDeployResponse(result);
       })
-      .catch((reason: string) => {
-        if (reason) {
-          vscode.window.showErrorMessage(reason, { title: 'Show output', action: 'SHOW_OUTPUT' }).then((m) => {
+      .catch((ex: any) => {
```

Comment:
because it is not always a reason.
It is an exception of type any instead of just a string.